go.mod

```diff
@@ -7,8 +7,11 @@ require (
 	github.com/stretchr/testify v1.8.4
 )
 
+require golang.org/x/net v0.15.0 // indirect
```

Comment:
You can place all the indirect dependencies in one require bock and all the direct dependencies in another. Not sure why it didn't do that automatically.